docs/REFERENCE.md

```diff
@@ -1023,31 +1023,37 @@ further parsing is done on the `note` field.
 
     {{ (index (lastpassRaw "SSH Private Key") 0).note }}
 
-### `onepassword` *uuid*
+### `onepassword` *uuid* [*vault uuid*]
```

Comment:
nit: please change `*vault uuid*` to `*vault-uuid*` (i.e. add a hyphen in the middle) so the new argument visually parses as a single token.

Reply:
Same comment in the use of `*vault uuid*` below.